Weather/src/app/interfaces/WeatherDataInterfaces.ts

```diff
@@ -0,0 +1,14 @@
+export interface Weather {
+    name: string;
+    temp: number;
+    description: string;
+    sys: {
+      country: string;
+    };
```

Comment:
For each property create a separate interface like -> `interface WeatherCountry {
{
      country: string;
    };
}`

and it will be like 
`export interface Weather {
    name: string;
    temp: number;
    description: string;
    sys: WeatherCountry .......`